src/test/gcp/cloudfunctions.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { expect } from "chai";
-import * as sinon from "sinon";
-import * as api from "../../api";
+import * as nock from "nock";
+
+import { functionsOrigin } from "../../api";
```

Comment:
nit* Either we don't need this import or all occurrence of `https://cloudfunctions.googleapis.com` should be replaced with it.

Reply:
Thanks.